lib/moog-require.js

```diff
@@ -231,7 +234,35 @@ module.exports = function(options) {
     }
     self.symlinksCache.set(type, link);
     return link;
-  };
+  }
+
+  function getDependencies({
+    dependencies = {},
+    devDependencies = {}
+  } = {}) {
+    return [
+      ...Object.keys(dependencies || {}),
+      ...Object.keys(devDependencies || {})
+    ];
+  }
+
+  function getWorkspacesDependencies(folder) {
+    return ({ workspaces = [] } = {}) => {
+      if (workspaces.length === 0) {
+        return [];
+      }
+
+      const pattern = workspaces.length === 1 ? workspaces[0] : `{${workspaces.join(',')}}`;
```

Comment:
Pretty sure the join call is always sufficient here

Reply:
I've added a comment, glob expects at least two entries when using curly braces. Silly bug!